src/main/java/io/token/banksample/Factory.java

```diff
@@ -43,7 +51,19 @@
                 config.fxAccounts(),
                 config.rejectAccounts(),
                 config.customerAccounts());
+
+        // By default, clock is set to UTC and expiration is set to 1 day
```

Comment:
my naive thought when reading this: if expiration is in terms of a duration, what does the clock's time zone matter? (or is this saying there can be trouble if something-system-ish isn't using UTC?)

Reply:
You're right, timezone doesn't matter, I will change the comment. It's used in case the user wants to put in a stopped clock for testing, or a centralized clock.